templates

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to AlBee-social!</title>
+</head>
+<body>
+    <div class="container">
+        <h2>Hi Utkarsh,</h2>
+        <p>Great to have you join <span class="highlight">AlBee-social!</span> <span class="emoji">🎉</span></p>
+        <p>We're excited to help you make your digital presence more professional.</p>
+        <p><a href="#" class="button">Verify Your Email</a></p>
+        <p>This link is valid for 24 hours.</p>
+        <p>Once you verify your email, you'll be directed straight to our system. It'll take less than a minute, and the process is super simple. 👍</p>
+        <p>Can't click the button? Copy and paste this link into your browser:</p>
+        <p><a href="http://116.202.210.102:5177/business-details/Sml5bmZBWW5rQj0+NDQ=" class="link">http://116.202.210.102:5177/business-details/Sml5bmZBWW5rQj0+NDQ=</a></p>

```

Comment:
instead of my link it should be your link (116.202.210.102) and name of client also change . also create a folder name templates/user_verification.html link